pages/index.vue

```diff
@@ -116,12 +107,15 @@
                     </div>
                 </div>
                 <div class="templates">
-                    <h2>Templates</h2>
-                    <p class="font-italic">Start by choosing a template</p>
+                    <h2>Layout</h2>
+                    <p class="font-italic">
+                        Start by choosing the layout of your comic. You can either create one panel in different sizes
```

Comment:
I we want to have a short text another option would be:

Choose your comic's layout. You can use a single panel or a multi-panel comic strip.